src/vs/editor/common/config/editorOptions.ts

```diff
@@ -3956,7 +3956,7 @@ export const enum EditorOption {
 export const EditorOptions = {
 	acceptSuggestionOnCommitCharacter: register(new EditorBooleanOption(
 		EditorOption.acceptSuggestionOnCommitCharacter, 'acceptSuggestionOnCommitCharacter', true,
-		{ markdownDescription: nls.localize('acceptSuggestionOnCommitCharacter', "Controls whether suggestions should be accepted on commit characters. For example, in JavaScript, the semi-colon (`;`) can be a commit character that accepts a suggestion and types that character.") }
+		{ markdownDescription: nls.localize('acceptSuggestionOnCommitCharacter', "Controls whether suggestions should be accepted on commit characters, including space (` `). For example, in JavaScript, the semi-colon (`;`) can be a commit character that accepts a suggestion and types that character.") }
```

Comment:
Suggestion to move that part into a separate sentence: "_Controls whether suggestions should be accepted on commit characters. For example, in JavaScript, the semi-colon (`;`) can be a commit character that accepts a suggestion and types that character. Note that space (` `) is a valid accept character too._"

Maybe also add a sample where space is used. I also curious where that happens